.github/actions/build-push/action.yaml

```diff
@@ -26,12 +31,16 @@ runs:
     with:
       images: ghcr.io/${{ github.repository }}
       flavor: latest=false
+      labels: |
+        org.opencontainers.image.created=${{ github.event.repository.updated_at}}
+        homepage=${{ github.server_url }}/${{ github.repository }}
```

Comment:
Why are you overwriting the labels here? The default behavior should be right.

Reply:
These two extra labels did not exist before, they are net-new.

Reply:
The metadata action should definitely set `org.opencontainers.image.created` and `homepage` is not an OCI image annotation (`org.opencontainers.image.url` is, and the metadata action should be setting that).

Reply:
Ok, I removed these flags. Perhaps we will remove `homepage` from goreleaser since it is redundant to `org.opencontainers.image.url`